src/main/kotlin/api/Door43ApiInterface.kt

```diff
@@ -0,0 +1,10 @@
+package api
+import retrofit2.http.GET
+import retrofit2.http.Query
+import io.reactivex.Observable
+
+interface Door43ApiInterface {
```

Comment:
Although I saw official kotlin doc using interface after interface name, I would suggest Door43ApiInterface to be Door43Api